Slave.cpp

```diff
@@ -338,6 +338,12 @@ void Slave::createTable(RelayLogInfo& rli,
 
     rli.setTable(tbl_name, db_name, std::move(table));
 
+    const auto key = std::make_pair(db_name, tbl_name);
+    auto it = m_struct_callbacks.find(key);
+    if (it != m_struct_callbacks.end()) {
+        const auto &t = rli.getTable(key);
```

Comment:
Why don't you just use pointer from `table`? Is it intended?

Reply:
Dereferencing the table pointer segfaults probably because of unique_ptr semantics but I'm not entirely sure. Calling rli.getTable seemed like a safer option.

Reply:
`rli.getTable` could potentially return an empty pointer which you don't check.
So, if you are afraid of using pointer from `table` (saved before `std::move(table)`), than you'd better check the result of `rli.getTable` before using it.